packages/dataviews/src/dataforms-layouts/panel/modal.tsx

```diff
@@ -143,11 +145,38 @@ function PanelModal< Item >( {
 				onClick={ () => setIsOpen( true ) }
 				disabled={ fieldDefinition.readOnly === true }
 				accessibleWhenDisabled
+				style={ summaryFields.length > 1 ? {
+					minHeight: 'auto',
+					height: 'auto',
+					alignItems: 'flex-start'
+				} : undefined }
 			>
-				<fieldDefinition.render
-					item={ data }
-					field={ fieldDefinition }
-				/>
+				{ summaryFields.length > 1 ? (
```

Comment:
This button component could be extracted as an utility to both Modal and Panel. SummaryButton, perhaps?